### PR TITLE
update acrobat reader

### PR DESCRIPTION
**Pull request by aronovgj**
update acrobat reader

### CODE REVIEW

File: automatic/adobereader/adobereader.nuspec [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
+
+Now includes support for installing English, Spanish, German, French or Japanese based on the LCID.</description>
+    <summary>Adobe Reader - View and interact with PDF files</summary>
+    <tags>adobereader pdf reader admin</tags>
   </metadata>
-  <files>
-    <file src="tools\**" target="tools" />
-  </files>
 </package>
-<!-- character encoding: “UTF-8” -->
```
Comment (ferventcoder) [inline automatic/adobereader/adobereader.nuspec]: We may have had some issues in translation here. No worries

File: automatic/adobereader/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -1,90 +1,7 @@
...
-  $locale = $mapping.Get_Item($LCID);
-  Write-Host "Switching to $locale"
-  $url = $url -replace 'en_US', "$locale"
-} else {
-  Write-Debug "Leaving the url to the default English version."
-  $url = $url
-}
-
-Install-ChocolateyPackage "$packageName" "$installerType" "$silentArgs" "$url" -validExitCodes $validExitCodes
+Install-ChocolateyPackage "$packageName" "$installerType" "$silentArgs" "$url"
```
Comment (ferventcoder) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: So none of this is required now?

**Comment by aronovgj** [comment]
this is changing the url depending on the system language. the idea was not to have to do this. that is why we're downloading the international installer which has all the languages in it. it should install the right one on its own. at least for me it did.

File: automatic/adobereader/tools/chocolateyInstall.ps1 (hunk 2 above)
Comment (aronovgj) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: this is changing the url depending on the system language. the idea was not to have to do this. that is why we're downloading the international installer which has all the languages in it. it should install the right one on its own. at least for me it did.

Comment (ferventcoder) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: ah right on.

Comment (aronovgj) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: note that you will have to change the download directory and remove the nopush variable in ketarin

**Pull request merged**